components/expenses/PayoutBankInformationForm.js

```diff
@@ -126,6 +126,7 @@ const Input = props => {
             >
               {() => (
                 <StyledSelect
+                  inputId="payout-bank-info"
```

Comment:
Since there can be multiple selects, won't we end up with multiple `input` having the same `id`?